### DISCUSSION TITLE
Rotation Tutor Overlay

### DISCUSSION THREAD

**Discussion by kittenblackfriends** [blish-hud/Blish-HUD#772] [Module Ideas]
A rotation Tutor Overlay takes an input of skills or key presses (options for either), and displays them on screen in a banner or marquee. UI elements should include the ability to generate a list, ability to select a list, and ability to resize the UI independently. Additional features would include the ability to create folders, remember the last active list for a character, and detect key presses to flash on the banner.

On top of this, a simple UI that counts to what auto attack stage you're at would be nice, as it can be hard to tell from a glance down or the animation.

**Comment by SeraphimSerapis** [comment]
As somebody who's pretty new to GW2 and the flashy combat, I'd love to see this functionality, too. It really helps to get a rotation into muscle memory before it eventually becomes less necessary.

**Comment by Taschenbuch** [comment]
Not sure if those implement this suggestion, but there are a few modules for skill rotations now
- https://blishhud.com/modules/?module=com.shooper.ddr
- https://blishhud.com/modules/?module=Nekres.RotationTrainer
